CloudM.Exercise.WebApi/Controllers/PackageController.cs

```diff
@@ -27,12 +31,63 @@ public async Task<PackageDependencies> Get(string name, string version)
                 PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
             });
 
+            var dependencies = npmPackage.Versions[version].Dependencies;
+
+            foreach (var dep in dependencies)
```

Comment:
First we need to check if the result from the dependencies extraction is different than null, otherwise we will attempt to loop through an empty collection which will lead to an exception.
Also we should use better naming, like 'department' instead of 'dep'